packages/webdriverio/tests/__mocks__/got.ts

```diff
@@ -218,7 +218,7 @@ const requestMock: any = jest.fn().mockImplementation((uri, params) => {
         const script = Function(params.json.script)
         let result
         script.call(this, ...params.json.args, (_result: any) => result = _result)
-        value = result ?? {}
+        value = result
```

Comment:
I believe this change causes the test to fail. It it really necessary?

Reply:
it is because this is a bug. Value cannot be undefined or null but the truth is that real call would return these values.

Reply:
then we need to adjust the unit test

Reply:
I don't know why it is failing and have trouble running tests locally. If you know how to solve it let me know otherwise I will close the PR and report this as an issue rather.